core/src/subgraph/instance_manager.rs

```diff
@@ -627,16 +627,17 @@ where
 
                 // Handle unexpected stream errors by marking the subgraph as failed.
                 Err(e) => {
+                    let message = format!("{:#}", e).replace("\n", " ");
```

Comment:
It might be nicer to replace `\n` with `\t` because then it is possible to get the nice formatting back by reversing the substitution (e.g., when you downloaded a log and want to study the backtrace in more detail)